config/application.rb

```diff
@@ -77,6 +77,20 @@ class Application < Rails::Application
     Dir["#{Rails.root}/config/routes/**/*.rb"].each do |route_file|
       config.paths['config/routes.rb'] << route_file
     end
+    
+    config.webpack = ActiveSupport::OrderedOptions.new
+    config.webpack.dev_server = ActiveSupport::OrderedOptions.new
+    
+    config.webpack.dev_server.host = proc { respond_to?(:request) ? request.host : 'localhost' }
+    config.webpack.dev_server.port = 3808
+    
+    config.webpack.dev_server.https = false
```

Comment:
I remember there used to be problems related to http/https when running in vagrant box. I currently run webpack with:

```
./node_modules/.bin/webpack-dev-server --config config/webpack.config.js --https --key /etc/pki/katello/private/foreman-devel.example.com-foreman-client.key --cert /etc/pki/katello/certs/foreman-devel.example.com-foreman-client.crt --cacert /etc/pki/katello/certs/katello-default-ca.crt --watch --progress --public foreman-devel.example.com --port 3808
```

and have
```yml
:webpack_dev_server: true
:webpack_dev_server_https: true
```
in config/settings.yaml